docs/examples/general/data_loading/numpy_reader.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def pipe1(file_root, file_filter):
</pre><p>For the sake of example, I think that those parameters should be hardcoded to <code>dali_dir_2d</code> and <code>"*.npy"</code> , respecitvely. Now it's a lot of definition chasing and it's not obvious at all that this pipeline loads 2D slices.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>